paddle/fluid/framework/details/execution_strategy.h

```diff
@@ -21,10 +21,15 @@ namespace details {
 
 struct ExecutionStrategy {
   enum ExecutorType { kDefault = 0, kExperimental = 1 };
+  enum UseDevice {
```

Comment:
Maybe DeviceType is more intuitive ？ And, it seems `kCPU` and `kXPU` make no difference in this PR?